crates/compiler/load_internal/src/file.rs

```diff
@@ -3838,6 +3838,36 @@ struct HeaderOutput<'a> {
     opt_platform_shorthand: Option<&'a str>,
 }
 
+fn ensure_roc_file<'a>(filename: &PathBuf, src_bytes: &[u8]) -> Result<(), LoadingProblem<'a>> {
+    match filename.extension() {
+        Some(ext) => {
+            let roc_ext = std::ffi::OsStr::new("roc");
```

Comment:
Is there a more canonical place I can get this string from?

Reply:
ROC_FILE_EXTENSION [exists in the current file](https://github.com/roc-lang/roc/blob/d62cc0b1d3e540a42bf58c4e3848c115f614a6c8/crates/compiler/load_internal/src/file.rs#L94).